api.md

```diff
@@ -255,6 +256,8 @@ still work on the resulting hypertable.
 | `associated_table_prefix` | Prefix for internal hypertable chunk names. Default is "_hyper". |
 | `migrate_data` | Set to `true` to migrate any existing `main_table` data to chunks in the new hypertable. A non-empty table will generate an error without this option. Note that, for large tables, the migration might take a long time. Defaults to false. |
 | `time_partitioning_func` | Function to convert incompatible primary time column values to compatible ones. The function must be `IMMUTABLE`. |
+| `replication_factor` | If non-zero, will create a distributed hypertable with each chunk being written to `replication_factor` different data nodes. The default value is 0. When creating a distributed hypertable, consider using [`create_distributed_hypertable`](#create_distributed_hypertable) in place of `create_hypertable`.  Note that values greater than 1 are not recommended in the current implementation.|
```

Comment:
The use of `replication_factor` in the description is confusing.  Consider "If non-zero, will create a distributed hypertable with each chunk being written to the number of data nodes corresponding to `replication_factor`"

Reply:
Changed as you suggested